packages/hadron-build/lib/target.js

```diff
@@ -450,6 +450,7 @@ class Target {
         outputDirectory: this.packagerOptions.out,
         exe: this.packagerOptions.name,
         name: this.productName,
+        shortcutName: this.productName,
```

Comment:
This is the fix for the shortcut made by .msi files.

Reply:
I am now unsure that the msi was ever doing the wrong thing, but this doesn't hurt - it at least makes it explicit.